src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -34,122 +34,135 @@
     }
 
-    public void onReload(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Reload");
+    public void onReload() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Reload");
     }
 
-    public void onDisable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Stopping");
+    public void onDisable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Stopping");
     }
 
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String[] args) {
-        if (label.equalsIgnoreCase("simple")){
+        if (label.equalsIgnoreCase("simple")) {
             if (sender instanceof Player) {
                 Player me = (Player) sender;
                 me.sendMessage("Plugin Working");
                 return true;
             }
-                if (label.equalsIgnoreCase("hawkeye")){//check command label
-                    if(args[0].equalsIgnoreCase("tnt")){//check if the second word is tnt
-                        if(sender instanceof Player){
-                            Player me =(Player)sender;
-                            PlayerInventory inventory = me.getInventory();
-                            for(int i = 0; i<inventory.getSize(); i++){
-                                ItemStack item = inventory.getItem(i);
-                                        if(item == null){
-                                            continue;
-                                        }
-                            if(isArrow(item)==true){
+            if (label.equalsIgnoreCase("hawkeye")) {//check command label
+                if (args[0].equalsIgnoreCase(ARROW_TYPE_TNT)) {//check if the second word is tnt
+                    if (sender instanceof Player) {
+                        Player me = (Player) sender;
+                        PlayerInventory inventory = me.getInventory();
+                        for (int i = 0; i < inventory.getSize(); i++) {
+                            ItemStack item = inventory.getItem(i);
+                            if (item == null) {
+                                continue;
+                            }
+                            if (isArrow(item) == true) {
                                 item.setType(Material.TIPPED_ARROW);
                                 item.setAmount(item.getAmount());
-                                PotionMeta potionMeta = (PotionMeta)item.getItemMeta();
+                                PotionMeta potionMeta = (PotionMeta) item.getItemMeta();
                                 potionMeta.setBasePotionData(new PotionData(PotionType.JUMP));
-                                int duration =10;
-                                int amplifier =10;
-                                boolean visible=true;
-                                potionMeta.addCustomEffect(new PotionEffect(PotionEffectType.CONDUIT_POWER,duration,amplifier),visible);
+                                int duration = 10;
```

Comment:
Issue #2) This line and line 99 say the same thing. Duplicate code should always be "refactored" https://en.wikipedia.org/wiki/Code_refactoring, Same for lines 76 and 77 and 101. You need a different technique here. Do you know one?